.eslintrc.json

```diff
@@ -43,36 +43,51 @@
     },
     { "files": ["**/__tests__/**"], "env": { "jest": true } },
     {
+      "extends": [
+        "plugin:@typescript-eslint/recommended",
+        "plugin:@typescript-eslint/stylistic"
+      ],
       "files": ["**/*.ts", "**/*.tsx"],
       "parser": "@typescript-eslint/parser",
       "parserOptions": {
-        "ecmaVersion": 2018,
         "sourceType": "module",
-        "ecmaFeatures": {
-          "jsx": true
-        },
```

Comment:
Unnecessary configuration line, I think _(are there odd situations I didn't notice?)_. Per https://typescript-eslint.io/packages/parser#jsx, although it defaults to `false`, the TypeScript compiler will always parse `.js`, `.mjs`, `.cjs`, `.jsx`, and `.tsx` files with `jsx: true`.

Reply:
What about `.mts` and `.cts`?

I think its best to keep `jsx: true` just in case. We don't want to introduce breaking changes in a patch release.

Reply:
> What about .mts and .cts?

From the same docs link: `.ts`, `.mts`, `.cts` files are always parsed as if this is false.

I'm curious: are you doing anything contrary to the default TypeScript behavior?

> I think its best to keep `jsx: true` just in case. We don't want to introduce breaking changes in a patch release.

I'm under the impression this file only impacts Next.js repo development, not end-user development. Just to make sure we're on the same page: am I wrong about that? If so, how would this impact end users?